R/R/install.R

```diff
@@ -0,0 +1,150 @@
+#' Install Metaflow Python package
+#'
+#' This function wraps installation functions from [reticulate][reticulate::reticulate] to install the Python packages
+#' **metaflow** and it's Python dependencies.
+#'
+#' This package uses the [reticulate][reticulate::reticulate] package
+#' to make an interface with the [Metaflow](https://metaflow.org/)
+#' Python package.
+#'
+#' @param method `character`, indicates to use `"conda"` or `"virtualenv"`.
+#' @param version `character`, version of Metaflow to install. The default version
+#' is the latest available on PyPi.
+#' @param ... other arguments sent to [reticulate::conda_install()] or
+#'    [reticulate::virtualenv_install()]
+#'
+#' @seealso
+#' [reticulate: Using reticulate in an R Package](https://rstudio.github.io/reticulate/articles/package.html),
+#' [reticulate: Installing Python Packages](https://rstudio.github.io/reticulate/articles/python_packages.html)
+#' @examples
+#' \dontrun{
+#' # not run because it requires Python
+#' install_metaflow()
+#' }
+#' @export
+install_metaflow <- function(method = c("conda", "virtualenv"),
```

Comment:
What is the expected behavior if `install_metaflow` is called multiple times?

Reply:
we don't allow `install_metaflow` with both conda & virtualenv. If the users runs `install_metaflow` with either conda/virtualenv multiple times, we simply re-install numpy & pandas & metaflow in the same environment.